python/lib/eeg.py

```diff
@@ -652,30 +666,88 @@ def fetch_and_insert_event_file(
             full_search = False,
         )
 
-        if not event_file:
+        if not event_data_file:
             message = "WARNING: no events file associated with " \
                       "physiological file ID " + str(physiological_file_id)
             print(message)
             return None
         else:
-            result = physiological.grep_event_from_physiological_file_id(
+            physiological_event_file_obj = PhysiologicalEventFile(self.db, self.verbose)
+            event_paths = physiological_event_file_obj.grep_event_paths_from_physiological_file_id(
                 physiological_file_id
             )
-            event_path = result[0]['FilePath'] if result else None
-            if not result:
-                event_data = utilities.read_tsv_file(event_file.path)
+
+            if not event_paths:
+                event_paths = []
+
+                event_data = utilities.read_tsv_file(event_data_file.path)
                 # copy the event file to the LORIS BIDS import directory
                 event_path = self.copy_file_to_loris_bids_dir(
-                    event_file.path, derivatives
+                    event_data_file.path, derivatives
                 )
                 # get the blake2b hash of the task events file
-                blake2 = blake2b(event_file.path.encode('utf-8')).hexdigest()
+                blake2 = blake2b(event_data_file.path.encode('utf-8')).hexdigest()
                 # insert event data in the database
                 physiological.insert_event_file(
                     event_data, event_path, physiological_file_id, blake2
                 )
 
-        return event_path
+                event_paths.extend([event_path])
+
+                # get events.json file and insert
+                # subject-specific metadata
+                event_metadata_file = self.bids_layout.get_nearest(
+                    event_data_file.path,
+                    return_type = 'tuple',
+                    strict = False,
+                    extension = 'json',
+                    suffix = 'events',
+                    all_ = False,
+                    full_search = False,
+                    subject=self.psc_id,
+                )
+                inheritance = False
+
+                if not event_metadata_file:
+                    # global events metadata
+                    event_metadata_file = self.bids_layout.get_nearest(
+                        event_data_file.path,
+                        return_type = 'tuple',
+                        strict = False,
+                        extension = 'json',
+                        suffix = 'events',
+                        all_ = False,
+                        full_search = False,
+                    )
+                    inheritance = True
+
+                    if not event_metadata_file:
+                        message = '\nWARNING: no events metadata files (event.json) associated' +/
```

Comment:
```suggestion
                        message = '\nWARNING: no events metadata files (event.json) associated' +
```

I think the above is what the line should be. There is an extra `/` that should not be there